app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -2,12 +2,10 @@
 
 <div class="component-markdown">
   <p>Components are packages of template, style, behaviour and documentation that live in your application.</p>
-  <p>A component must:</p>
   <ul>
-    <li><a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_principles.md">meet the component principles</a></li>
-    <li><a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md">follow component conventions</a></li>
+    <li><a href="https://docs.publishing.service.gov.uk/manual/components.html">all documentation for components</a></li>
```

Comment:
This feels valuable but seems a bit abrupt. I think a statement that also point to the repo, as we used to have is more natural ("See the govuk_publishing_components gem for further details, or a list of all component guides.")

Reply:
Yeah, I agree. I'll restore the original wording, thanks.